bson/primitive/objectid.go

```diff
@@ -73,15 +73,15 @@ func (id ObjectID) IsZero() bool {
 // ObjectIDFromHex creates a new ObjectID from a hex string. It returns an error if the hex string is not a
 // valid ObjectID.
 func ObjectIDFromHex(s string) (ObjectID, error) {
+	if len(s) != 24 {
+		return NilObjectID, ErrInvalidHex
+	}
+	
```

Comment:
Can you run gofmt? There is an extra tab here.

Reply:
Done